starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -64,6 +67,9 @@ async def syncing() -> Union[dict, bool]:
 
 
 # pylint: disable=too-many-arguments
+# Events response does not currently conform to RPC specs
+# and will need fixing before validation is added
+# @validate_schema("getEvents")
 async def get_events(
```

Comment:
This will have to be addressed in a separate PR

Reply:
This is the way. Not everything has to be ready from the start.

Reply:
Yup, we should add an issue in the `devnet` repo after PR is published